android/app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
We should never add google-services.json to the git repo

Reply:
will remove it